app/scripts/store/dimStoreItem.directive.js

```diff
@@ -60,6 +60,7 @@
 
       vm.clicked = function openPopup(item, e) {
         e.stopPropagation();
+        debugger
```

Comment:
absolutely not! that was totally intentional......

but you know... I've gone ahead and removed it anyway.

fixed